cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -283,31 +283,42 @@ var __AdyenHelper : Object = {
 
     createCardObjectSF: function (isOneClick, formType) {
         var card;
+        var cardType = "scheme";
```

Comment:
see below some changes regarding cards could you explain what it does and why it included into the ideal component PR ?